src/manifold.cpp

```diff
@@ -514,22 +514,6 @@ size_t Manifold::NumDegenerateTris() const {
   return GetCsgLeafNode().GetImpl()->NumDegenerateTris();
 }
 
-/**
- * This is a checksum-style verification of the collider, simply returning the
- * total number of edge-face bounding box overlaps between this and other.
- *
- * @param other A Manifold to overlap with.
- */
-size_t Manifold::NumOverlaps(const Manifold& other) const {
```

Comment:
I just removed this function because it is not used, and to be honest, I don't think the number of overlaps from the collider result is meaningful.